### PR TITLE
Feat [8694jyx60] add the logic if a potential friend havent liked me yet

### PR DESCRIPTION
**Pull request by Svitlana-G**

### CODE REVIEW

File: src/pages/MessagesAndFriends.tsx [hunk 1]
```diff
@@ -111,12 +132,14 @@ const MessagesAndFriends = () => {
             />
           </Box>
         )}
-        <Match
-          isMatchModalOpen={isMatchModalOpen}
-          onClose={handleMatchClose}
-          onChat={startChat}
-          friendsAvatar={friendsData.photo[0]?.src}
-        />
+        {isMatchModalOpen && (
```
Comment (LyubovSugak) [inline src/pages/MessagesAndFriends.tsx:135]: you don't need condition here, you passed prop

File: src/pages/MessagesAndFriends.tsx [hunk 2]
```diff
@@ -70,10 +77,24 @@ const MessagesAndFriends = () => {
     goToNextPotentialFriend(currentPotentialFriend)
   }
 
+  const isLiked = (accountId: string, likedUsersArray: string[]): boolean => {
+    const likedMe = likedUsersArray.includes(accountId)
+    if (likedMe) {
```
Comment (LyubovSugak) [inline src/pages/MessagesAndFriends.tsx:82]: I would recommend not to use such conditions, it's like red flag - means smth definitely wrong. 
`const isLiked = (accountId: string, likedUsersArray: string[]): boolean => {
          return likedUsersArray.includes(accountId)
  }`

**Review by LyubovSugak — CHANGES_REQUESTED** [review]

File: src/pages/MessagesAndFriends.tsx (hunk 2 above)
Comment (Svitlana-G) [reply src/pages/MessagesAndFriends.tsx]: Thanks. Fixed it.

File: src/pages/MessagesAndFriends.tsx (hunk 1 above)
Comment (Svitlana-G) [reply src/pages/MessagesAndFriends.tsx]: Removed

**Review by LyubovSugak — APPROVED** [review]

**Pull request merged**